Tests/GRPCTests/ConnectionManagerTests.swift

```diff
@@ -1412,6 +1418,54 @@ extension ConnectionManagerTests {
       XCTAssert(error is DoomedChannelError)
     }
   }
+
+  #if canImport(Network)
+  func testDefaultChannelProvider_NWParametersConfigurator() throws {
+    let counter = NIOLockedValueBox(0)
+    let group = NIOTSEventLoopGroup(loopCount: 1)
```

Comment:
The shutdown should be done in a `defer` block just after this or just use the singleton.